ENTITY/HasEntityClearLosToEntityAdjustForCover.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: ENTITY
+aliases: ["0x394BDE2A7BBA031E", "_HAS_ENTITY_CLEAR_LOS_TO_ENTITY_2"]
+---
+## HAS_ENTITY_CLEAR_LOS_TO_ENTITY_ADJUST_FOR_COVER
+
+```c
+// 0x394BDE2A7BBA031E
+cs_type(Any) BOOL HAS_ENTITY_CLEAR_LOS_TO_ENTITY_ADJUST_FOR_COVER(Entity entity1, Entity entity2, int traceType);
+```
+
```

Comment:
The NativeDB introduced comments are somewhat useful, they're automatically added by the importer for natives and they allow you to know in which game build a native was added. I would suggest preserving them.

```
NativeDB Introduced: v1868
```